Fixes part of #1367: Remove the boilerplate "isTablet" if statements from the code (ProfileChooser Part)

MohamedMedhat1998 opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
<!--
  - Explain what your PR does. If this PR fixes an existing bug, please include
  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
  - when this PR is merged.
  -->
- Refactored `ProfileChooserFragmentPresenter` code by removing the boilerplate `if` statements that check the type of the device and replacing them with XML values overridden in different configurations.  
- Fixes #1367 
## Checklist
<!-- Please tick the relevant boxes by putting an "x" in them. -->
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

MohamedMedhat1998 committed:
-Fixed "isTablet" boilerplate (ProfileChooser part)

rt4914 commented on app/src/main/res/values-xhdpi/dimens.xml:
```diff
@@ -2,5 +2,4 @@
 <resources>
   <dimen name="enter_text_view_padding_start">120dp</dimen>
   <dimen name="enter_text_view_padding_end">120dp</dimen>
-  <integer name="profile_chooser_span_count">5</integer>
```
Why was this removed?

MohamedMedhat1998 replied:
It was moved to instead `values-land-xhdpi/dimens.xml` of `values-xhdpi/dimens.xml` because we only update the span count based on the density in the landscape mode only.

rt4914 commented on app/src/main/res/values/dimens.xml:
```diff
@@ -6,7 +6,8 @@
   <integer name="fade_in_duration_ms">1000</integer>
   <integer name="fade_out_duration_ms">1000</integer>
-  <integer name="profile_chooser_span_count">4</integer>
+  <integer name="profile_chooser_span_count">2</integer>
```
Transfer all integers to `integer.xml` file for this PR and previous PRs too.

MohamedMedhat1998 replied:
Ok, I'll do it now

MohamedMedhat1998 replied:
Done

rt4914 reviewed (changes requested):
One of the major change in this PR is to shift the `integer` values to `integer.xml` file instead of `dimens.xml` file. And this applies to your other new/old PRs too.

MohamedMedhat1998 committed:
-Merged with the develop and solved the conflicts
-Moved all integer values from "dimens.xml" to "integers.xml" taking the different configurations in consideration

MohamedMedhat1998 commented:
@rt4914 PTAL  
- I have moved the integer values from `dimens.xml` to their proper files in `integers.xml`  
- I have updated the description of the PR to be `Fixes #1367` instead of `Fixes part of #1367`

rt4914 commented on app/src/main/res/values/dimens.xml:
```diff
@@ -3,10 +3,6 @@
   <dimen name="activity_vertical_margin">16dp</dimen>
-
-  <integer name="fade_in_duration_ms">1000</integer>
-  <integer name="fade_out_duration_ms">1000</integer>
```
`fade_in` , `fade_out` you have shifted but you have not updated the implementation code where they are getting used and maybe there are not even getting used. So to be on save side, I suggest shifting it back to its original so that it does not create any issues for others.

MohamedMedhat1998 replied:
Ok, I'll update it

rt4914 reviewed (approved):
LGTM, just one revert change.

MohamedMedhat1998 committed:
-moved "fade_in" and "fade_out" values back to "dimens.xml" to avoid confusion and issues for the others
